prime-dotnet-webapi/ViewModels/Enrollee/SelfDeclarationViewModel.cs

```diff
@@ -11,5 +11,6 @@ public class SelfDeclarationViewModel
         public int SelfDeclarationTypeCode { get; set; }
         public string SelfDeclarationDetails { get; set; }
         public IEnumerable<Guid> DocumentGuids { get; set; }
+        public bool IsHidden { get; set; }
```

Comment:
Do you need this on the View Model? It looks like you don't. Also, if you _do_ need it, it should also be `Hidden`.

Reply:
No, I swear I remember deleting it from VM lol